plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/IdentifierVerifier.java

```diff
@@ -87,4 +92,20 @@ public static Optional<String> verifyPackageName(final String packageName) {
 						.format(Messages.IdentifierVerifier_PackageNameMessage, message, packageName))
 				.findFirst();
 	}
+
+	public static Optional<String> verifyQualifiedIdentifier(final String qualifiedName) {
+		if (qualifiedName == null || qualifiedName.isBlank() || qualifiedName.startsWith(SEPARATION_POINT)
+				|| qualifiedName.endsWith(SEPARATION_POINT) || qualifiedName.contains("..")) { //$NON-NLS-1$
+			return Optional
+					.of(MessageFormat.format(Messages.IdentifierVerifier_NameNotAValidIdentifier, qualifiedName));
+		}
+		return verifyQualifiedByDelimiter(qualifiedName, "\\."); //$NON-NLS-1$
+	}
+
+	private static Optional<String> verifyQualifiedByDelimiter(final String name, final String delimiterRegex) {
+		return Stream.of(name.split(delimiterRegex, -1)).map(seg -> IdentifierVerifier.verifyIdentifier(seg, null))
```

Comment:
I believe this can be simplified to:
```suggestion
		return Stream.of(name.split(delimiterRegex, -1)).map(IdentifierVerifier::verifyIdentifier)
```